src/D2L.CodeStyle.Annotations/D2L.CodeStyle.Annotations.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net20;netstandard2.0</TargetFrameworks>
-    <LangVersion>7.3</LangVersion>
+    <!-- We build this for .NET 2.0 and the implicit usings bring in namespaces that didn't exist back then -->
+    <ImplicitUsings>disable</ImplicitUsings>

```

Comment:
.NET 2.0 was for the COM stuff, I'm not sure if we still need to keep doing this but whatever, its just the annotations DLL.